### PR TITLE
feature:category filter added

### PR DESCRIPTION
**Pull request by abhishektanwar**
FEATURE: ADDED CATEGORY FILTER

FILES TO CHECK
- src/Reducers/videosReducer.js
- src/components/CategoryCard/index.jsx
- src/components/FeaturedContainer/index.jsx
- src/helpers/helpers.js
- src/screens/ExploreVideos/index.jsx

### CODE REVIEW

**Review by sach10-create — APPROVED** [review]
Lots of commented code.
Can be removed.
rest code looks clean

File: src/Contexts/VideoListingContext/index.jsx [hunk 1]
```diff
@@ -52,7 +53,8 @@ const VideoListingProvider = (props) => {
         if(response.status === 200){
           const categories = response.data.categories;
           const categoriesFilterData = categories.reduce(
-            (acc, item) => ({ ...acc, [item.category]: false }),
+            (acc, item) => ({ ...acc,[item.category]:{"isSelected": false,"title":item.title} }),
+            // (acc,item) => acc.push({[item.category]: false,"title":[item.title]}),
             {}
```
Comment (srishtigautam21) [inline src/Contexts/VideoListingContext/index.jsx:58]: remove the commented code

File: src/components/CategoryCard/index.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  // videoListingDispatch({
+  //   type: CATEGORY_FILTER,
+  //   payload: {
+  //     category: {
+  //       key: key,
+  //       value: { ...value, ["isSelected"]: !value.isSelected },
+  //     },
+  //   },
+  // })
+  return (
```
Comment (srishtigautam21) [inline src/components/CategoryCard/index.jsx:20]: remove commented code

**Review by srishtigautam21 — APPROVED** [review]
UI suggestion:
The explore button on landingPage is not working
Add a favicon 
Appreciation:
The video listing page looks awesome all the functionality work fine
Good work Good to merge

**Pull request merged**